Formula/f/flang.rb

```diff
@@ -100,20 +86,13 @@ def install
     end
   end
 
-  def caveats
-    <<~EOS
-      Homebrew LLVM is built with LLVM_ENABLE_EH=ON, so binaries built by `#{flang_driver}`
-      require linkage to the C++ standard library. `#{flang_driver}` is configured to do this
-      automatically.
-    EOS
-  end
-
   test do
     # FIXME: We should remove the two variables below from the environment
     #        to test that `flang` can find our config files correctly, but
     #        this seems to break CI (but can't be reproduced locally).
     # ENV.delete "CPATH"
     # ENV.delete "SDKROOT"
+    assert_match "Configuration file: #{etc}/clang/", shell_output("#{bin/flang_driver} --version")
```

Comment:
I think we are hitting https://github.com/Homebrew/homebrew-test-bot/issues/805. Locally, don't see an issue
```console
❯ flang-new --version | grep Config
Configuration file: /opt/homebrew/etc/clang/arm64-apple-darwin24.cfg
```

Will remove test for next run.